coreos_prometheus/prometheus/prometheus-prometheus.yaml

```diff
@@ -1,49 +0,0 @@
-apiVersion: monitoring.coreos.com/v1
```

Comment:
did you intend to remove this? 

looks like it's still in the most recent version - https://github.com/prometheus-operator/kube-prometheus/blob/e02554298cb62b5533f3407c8eacc664e80bc74b/manifests/prometheus-prometheus.yaml#L1